include/z64player.h

```diff
@@ -52,7 +52,7 @@ typedef enum {
 } PlayerEnvHazard;
 
 typedef enum PlayerItemAction {
-    /*   -1 */ PLAYER_IA_MINUS1 = -1,
+    /*   -1 */ PLAYER_IA_CONTINUE = -1, // TODO: determine usages with more player docs, possibly split into seperate values
```

Comment:
The name `CONTINUE` definitely wouldn't make sense in OoT since MM decided to merge the itemAction and ExchangeItem systems, and this applies more for the exchangeItemAction usages of `-1`. There are other uses of `-1` like it being connected to where it remembers what the previous item was when you're holding shield (with R) eg. if you have hookshot in hand and you shield and let go, you still have hookshot in hand. So it's a value that has different meanings in different contexts.

We recently had 2 defines for -1, but decided to keep it to 1 until more is understood, see the thread convo here:
https://discord.com/channels/688807550715560050/1124540483591491684/1125074160889114644

Reply:
That said, I think I would prefer to keep it generally named instead of naming it after only 1 specific purpose when it has other purposes too

Reply:
Yeah the usage with shield and returning to previous item is what I know it to do from oot. but yeah if its different here, then fair enough

Reply:
The -1 itemAction still have that same shielding/return to previous item functionality in MM it seems, they just made it multipurpose

Reply:
> That said, I think I would prefer to keep it generally named instead of naming it after only 1 specific purpose when it has other purposes too

I'll revert it back to `PLAYER_IA_MINUS1` for the time being. I also plan on adding a comment just above the enum (it is rather too big to put it directly next to the value) explain the 3 current known usages of it.